tests/Api/Admin/PaymentMethodsTest.php

```diff
@@ -0,0 +1,173 @@
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        $this->client->request(method: 'GET', uri: '/api/v2/admin/payment-methods', server: $header);
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/payment_method/get_payment_methods_response',
+            Response::HTTP_OK,
+        );
+    }
+
+    /** @test */
+    public function it_creates_a_payment_method(): void
+    {
+        $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'channel.yaml',
+        ]);
+
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        $this->client->request('POST', '/api/v2/admin/payment-methods', [], [], $header, json_encode([
+            'code' => 'test',
+            'name' => 'Test',
+            'description' => 'Test',
+            'translations' => [
+                'en_US' => [
+                    'name' => 'Test',
+                    'description' => 'Test',
+                    'instructions' => 'Testing instructions',
+                ],
+            ],
+            'gatewayConfig' => [
+                'factoryName' => 'paypal_express_checkout',
+                'gatewayName' => 'paypal_express_checkout',
+                'config' => [
+                    'username' => 'test',
+                    'password' => 'test',
+                    'signature' => 'test',
+                    'sandbox' => true,
+                ],
+            ],
+            'enabled' => true,
+        ]));
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/payment_method/create_payment_method_response',
+            Response::HTTP_CREATED,
+        );
+    }
+
+    /** @test */
+    public function it_updates_a_payment_method(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'channel.yaml',
+            'payment_method.yaml',
+        ]);
+
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        /** @var PaymentMethodInterface $paymentMethod */
+        $paymentMethod = $fixtures['paypal_payment_method'];
+
+        $this->client->request(
+            method: 'GET',
+            uri: sprintf('/api/v2/admin/payment-methods/%s/gateway-config', $paymentMethod->getCode()),
+            server: $header,
+        );
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/payment_method/get_payment_method_gateway_config_before_update_response',
+            Response::HTTP_OK,
+        );
+
+        $this->client->request(
+            method: 'PUT',
+            uri: sprintf('/api/v2/admin/payment-methods/%s', $paymentMethod->getCode()),
+            server: $header,
+            content: json_encode([
+                'gatewayConfig' => [
```

Comment:
Update all possible fields of payment method, like `translations`, `channels`, etc